index.html

```diff
@@ -42,106 +49,115 @@ <h1><span class="h-span">I’m a software developer</span></h1>
             </button>
         </div>
     </section>
+</header>
+<!--Header Ends Here-->
+<!--Work card content starts Here-->
     <section class="work-card">
         <h1 class="myrecentwork">My Recent Works</h1>
         <img class="ind" src="images/Indicator.png" alt="">
-        <div class="work-container">
-            <div class="img-placer">
-                <img src="images/im1.png" alt="">
+        <div class="work-container-card">
+            <div class="work-container">
+                <div class="img-placer">
+                    <img src="images/im1.png" alt="">
+                </div>
+                <div class="txt-info">
+                    <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="languages">
+                        <li><a href="">Ruby on rails</a></li>
+                        <li><a href="">css</a></li>
+                        <li><a href="">JavaScript</a></li>
+                        <li><a href="">html</a></li>
+                    </ul>
```

Comment:
- [ ] [OPTIONAL] Please can you try to modify the languages's font, color, size and alignment to match to that of figma design, I will make your webpage more attractive :+1: :+1: 

**Your design** | **Figma design**
| --- | ---|
![image](https://user-images.githubusercontent.com/102757126/199717176-25cded68-f0be-4b33-ab88-ddfe5de1cd4b.png) | ![image](https://user-images.githubusercontent.com/102757126/199717320-fa06db17-9ef1-4f2d-9288-41d221bfe471.png)